src/react/projects/spark-core-react/src/SprkDictionary/SprkDictionary.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const SprkDictionary = (props) => {
+    const { keyValuePairs, variant, idString, additionalClasses, ...other } = props;
+    const classNames = classnames(
+        'sprk-c-Dictionary',
+        additionalClasses, {
+            'sprk-c-Dictionary--striped': variant === 'striped'
+        },
+    );
+
+    const keyValuePairsMap = Object.keys(keyValuePairs).map(function (term) {
```

Comment:
Hey RV, I'd like to see a single parameter ES6 arrow function for the map callback to keep our writing styles consistent.